js/admin/carrier_wizard.js

```diff
@@ -403,6 +400,13 @@ function bind_inputs()
 			return false;
 	});
 
+	$('tr.range_sup td input:text, tr.range_inf td input:text').blur(function () {
```

Comment:
This selector is written at least twice elsewhere (lines 410 and 569). Please consider writing an encapsulating function somewhere else:
```js
var carrier = {
  getRangeBoundaryInputs: function() {
     return $('tr.range_sup td input:text, tr.range_inf td input:text');
  }
};
```
and then
```js
var rangeBoundaryInputs = carrier.getRangeBoundaryInputs();
rangeBoundaryInputs
  .blur(...)
  .typeWatch(...);